cuenca_validations/types/general.py

```diff
@@ -9,10 +15,39 @@ def __init__(self, *args, **kwargs):
         sanitize_dict(self)
 
 
+class CJSONEncoder(json.JSONEncoder):
```

Comment:
Change class name to `JSONEncoder`. cjson is a commonly used library, and this class name makes it sound like it uses that library